### PR TITLE
[ UI ] - Popup component MUI version updated.

### LINKED ISSUE #12423: [UI] Migrate Popup Component to use MUI v5 Styling and Sistent Component
### Current Behavior
Meshery's Popup.js component is using an outdated version of Material-UI (v4), which leads to increased maintenance and performance issues.

### Desired Behavior
[x] use [Sistent](https://github.com/layer5io/sistent) components for MUI v4 to MUI v5 migration

Sistent is our design system that uses MUI v5, you can also refer to this [doc](https://docs.meshery.io/project/contributing/contributing-ui-sistent) to understand how to use Sistent.

### Acceptance Tests
The component must be updated without introducing any breaking changes in functionality or theming across all instances where it is used.

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Pratik-050**
**Notes for Reviewers**

This PR fixes #12423

Migrated the Popup.js function to MUI V5 using supported sistent components.

Light mode:
Previous component:
Screenshot from 2024-10-25 00-27-37
New component:
Screenshot from 2024-10-25 00-27-22

Dark mode:
Previous component:
Screenshot from 2024-10-25 00-28-15
New component:
Screenshot from 2024-10-25 00-28-16

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by Pratik-050** [comment]
Hello @SAHU-01 @yash37158 @sudhanshutech I've made the PR as assigned to me have a look at this. Thanks!

**Comment by vishalvivekm** [comment]
@Pratik-050 
Thank you for your contributions! Let's discuss this during the Meshery Development call today at 9:00 AM CT | 7:30 PM IST. Adding it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#) || [meeting link](https://meet.layer5.io/meshery).

**Comment by SAHU-01** [comment]
This looks good to me @Pratik-050, have you run this locally to test with other components, to ensure it's not breaking the existing functionality/style? 
Once, you add the necessary sistent token changes we should be good to merge.

**Comment by Pratik-050** [comment]
Hello @SAHU-01 everything is working fine, I'm just unable to find the correct background token from sistent. 
Once that is fixed then we are good to go. Can you help me finding that?

**Comment by Pratik-050** [comment]
Hello @SAHU-01 I've made some some changes please have a look. @leecalcote @yash37158 @sudhanshutech does the new colors good to go?

**Comment by leecalcote** [comment]
Is there a pop-up component in Sistent?

**Review by leecalcote — APPROVED** [review]

**Comment by Pratik-050** [comment]
> Is there a pop-up component in Sistent?

No sir I don't think there's a dedicated component for popup in the sistent library.

File: ui/components/Popup.js [hunk 1]
```diff
@@ -1,76 +1,60 @@
-import { Button, Grid, IconButton, Typography, useTheme } from '@material-ui/core';
-import { makeStyles } from '@material-ui/core/styles';
-import CloseIcon from '@material-ui/icons/Close';
+import { Button, Grid, IconButton, Typography, styled } from '@layer5/sistent';
+import { UsesSistent } from './SistentWrapper';
+import { useTheme } from '@material-ui/core';
```
Comment (sudhanshutech) [inline ui/components/Popup.js:3]: this can also be used from sistent

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Comment by sudhanshutech** [comment]
LGTM otherwise

**Comment by sudhanshutech** [comment]
LGTM otherwise

**Review by sudhanshutech — APPROVED** [review]
// @SAHU-01 fyi

**Pull request merged**